mp/src/game/client/momentum/ui/HUD/hud_mapfinished.cpp

```diff
@@ -406,6 +429,8 @@ void CHudMapFinishedDialog::SetCurrentPage(int pageNum)
         ANSI_TO_UNICODE(ansiTime, unicodeTime);
         m_pZoneOverallTime->SetText(CConstructLocalizedString(m_pwZoneTime, unicodeTime));//"Zone time" (time for that zone)
 
+        SetComparisonForPage(m_iCurrentPage);
```

Comment:
Since this happens now for both branches you can pull this out to above the `if (m_iCurrentPage < 1)` check.